package.json

```diff
@@ -42,6 +42,8 @@
     "@types/lodash-es": "4.17.12",
     "@types/sdp-transform": "2.4.5",
     "@types/strophe": "1.2.37",
+    "@typescript-eslint/eslint-plugin": "8.33.1",
```

Comment:
We should already be transitively depending on these, see above a potential way to avoid depending on them directly.